apps/ima/fields/index.js

```diff
@@ -6,5 +6,79 @@ module.exports = {
     mixin: 'radio-group',
     options: ['person-named', 'has-legal-representative', 'someone-else'],
     validate: 'required'
+  },
+  'does-exception-apply': {
+    isPageHeading: false,
+    mixin: 'radio-group',
+    options: [{
+      value: 'yes',
+      toggle: 'exception-details-fieldset',
+      child: 'partials/exception-apply'
+    }, {
+      value: 'no'
+    }],
+    validate: 'required'
+  },
+  'does-exception-apply-detail': {
+    labelClassName: 'bold',
+    mixin: 'textarea',
+    attributes: [{
+      attribute: 'rows',
+      value: 5
+    }],
+    validate: ['required', 'notUrl', { type: 'regex', arguments: /^[^\[\]\|<>]{1,15000}$/ }],
```

Comment:
I think you discussed the separating the maxlength and regex validations with Sulthan? Can you make those changes here as well?